<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>331</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Название рецепта</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ингредиенты</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Рецепт</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Название файла фотографии</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="photoname">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>320</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="recipe">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>261</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="ingred">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>261</width>
     <height>91</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Номера ингредиентов(из добавленных) через пробел</string>
   </property>
  </widget>
  <widget class="QLabel" name="add_recipe_error">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
